libdnf/dnf-goal.cpp

```diff
@@ -183,17 +183,17 @@ void
 dnf_goal_add_protected(HyGoal goal, DnfPackageSet *pset)
 {
     Pool *pool = dnf_sack_get_pool(goal->sack);
-    Map *protected = goal->protected;
+    Map *protected_pkgs = goal->protected_pkgs;
```

Comment:
I'd be happier if the the rename was part of a different pull request, but keep it as it is.